lib/scarpe/wv/shape.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative "shape_helper"
+
+class Scarpe
+  class WebviewShape < Scarpe::WebviewWidget
+    include ShapeHelper
+
+    def initialize(properties)
+      super(properties)
+    end
+
+    def element(&block)
+      HTML.render do |h|
+        h.div(id: html_id, style: style) do
+          h.svg(width: "400", height: "500") do
+            h.path(d: shape_path, style: "stroke:#{stroke_color};stroke-width:2")
+          end
+          block.call(h) if block_given?
+        end
+      end
+    end
+
+    private
+
+    def style
+      {
+        width: "400",
+        height: "900",
+      }
+    end
+
+    def stroke_color
```

Comment:
It feels like this should be configurable. But I won't worry more about it for now. If I care that much I can add that feature later :-)